src/WebServer.UnitTests/Rest/RestRouteHandlerTests_UriFormatValidation.cs

```diff
@@ -0,0 +1,105 @@
+﻿using Restup.HttpMessage.Models.Schemas;
+using Restup.Webserver.Attributes;
+using Restup.Webserver.Models.Contracts;
+using Restup.Webserver.Models.Schemas;
+using Restup.Webserver.Rest;
+using Restup.Webserver.UnitTests.TestHelpers;
+using System;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Restup.Webserver.UnitTests.Rest
+{
+    [TestClass]
+    public class RestRouteHandlerTests_UriFormatValidation
```

Comment:
Format of the other test classes is with a . instead of the _ , I know, "mieren*&^%"